translations/en-us.yml

```diff
@@ -1907,18 +1907,30 @@ osf-components:
             retry: 'retry delete'
         help_modal:
             heading: 'Using the OSF Files Browser'
+            providers: 'See All Files in a Provider'
+            providers_detail: 'All connected storage providers are displayed in the left navbar. Choose one provider to view contents.'
+            select: 'Select Files/Folders'
+            select_detail: 'Click on a row (outside of the file or folder name) to show further actions in the top toolbar. Use Command or Shift keys to select multiple files.'
```

Comment:
I put a comment on the ticket just now that you don't need to command click to select multiple files. This bit of help could use a little re-write.

Reply:
Right, I noticed that this doesn't mention rename and delete either, so I will put this on hold until we decide what we want to add to this modal